src/components/Book.jsx

```diff
@@ -0,0 +1,10 @@
+const Book = () => (
+  <div>
+    <h2>Add abook</h2>
+    <ul>
+      <li>Music</li>
+    </ul>
+  </div>
+);
+
+export default Book;
```

Comment:
- [ ] The Book component representing a single book should take in the book data as a prop and render it. Let's do that. Just as we have passed in the title and author as a prop earlier in the Books component, we can then accept the props here and render the title and author. In the end you should have something like this:

  ```
  import PropTypes from 'prop-types';
  import Button from './shared/Button';
  
  const Book = (props) => {
    const { title, author } = props;
    return (
      <div>
        <h2>{title}</h2>
        <ul>
          <li>{author}</li>
        </ul>
        <Button value="remove" />
      </div>
    );
  };
  
  Book.propTypes = {
    title: PropTypes.string.isRequired,
    author: PropTypes.string.isRequired,
  };

  export default Book;
  ```

  **NB:** You may need to install the `prop-types` package for prop validation and type checking